walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/requirements/IdTokenRequirement.kt

```diff
@@ -24,7 +24,7 @@ class IdTokenRequirement(
     val scope: String,
 
     // Nonce is generated using user DID.
-    val nonce: String,
+    private var _nonce: String,
```

Comment:
why prefix nonce with _? Is this a kotlin convention?

Reply:
There are two "nonce" variables. the private "_nonce" holds the value, while the getter for nonce and theoretically a setter but in this case a function can do additional checks.